modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Inspecting encapsulated namespaces
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+An administrator logged in to the host OS wishing to obtain information about the Kubernetes system for debugging or auditing purposes will need to be aware of the new mount namespace. A shell originating in a container, such as `oc debug` will already be inside the Kubernetes namespace. A shell originating in an SSH session will be in the default namespace, and an administrator will need to run the `kubensenter` script as root to view or interact with these mountpoints.
+The `kubensenter` script is aware of the state of the mount encapsulation feature status, and is safe to run even if encapsulation is not enabled. In that case it will simply execute the shell or requested command in the default mount namespace.
+
+. Open a remote shell to the cluster host or `oc debug node/<node-name>`.
+
+. To run a single command inside the Kubernetes namespace, provide the command and any arguments to the `kubensenter` script.  For example, to run the `findmnt` command inside the Kubernetes namespace, issue the following command:
+[source,terminal]
+----
+$ sudo kubensenter findmnt
+----
+
+.Example output
+[source,terminal]
+----
+  kubensenter: Autodetect: kubens.service namespace found at /run/kubens/mnt
+  TARGET                                                                                                                                                   SOURCE                 FSTYPE     OPTIONS
+  /                                                                                                                                                        /dev/sda4[/ostree/deploy/rhcos/deploy/32074f0e8e5ec453e56f5a8a7bc9347eaa4172349ceab9c22b709d9d71a3f4b0.0]
+  |                                                                                                                                                                               xfs        rw,relatime,seclabel,attr2,inode64,logbufs=8,logbsize=32k,prjquota
+                             shm                    tmpfs
+  ...
+----
+
+To start a new interactive shell inside the Kubernetes namespace, run the `kubensenter` script without any arguments:
+[source,terminal]
+----
+$ sudo kubensenter
```

Comment:
If we are running inside the cluster using `oc debug node`, the bash prompt will look like this:

```suggestion
sh-4.4# sudo kubensenter
```